client.bal

```diff
@@ -0,0 +1,396 @@
+        }
+        if (orderBy is string){
+            optional.orderBy = orderBy;
+        }
+        return getFiles(self.httpClient, optional);
+    }
+
+    # Retrieve files by Name
+    # 
+    # + fileName - Name of the file to search (Partial search)
+    # + noOfFiles - Number of files to retreive 
+    # + orderBy - A comma-separated list of sort keys. Valid keys are 'createdTime', 'folder', 'modifiedByMeTime', 
+    #             'modifiedTime', 'name', 'name_natural', 'quotaBytesUsed', 'recency', 'sharedWithMeTime', 'starred', 
+    #              and 'viewedByMeTime'
+    # + return - If successful, returns stream of files `stream<File>`. Else returns `error`
+    remote function getFilesByName(string fileName, int? noOfFiles = (), string? orderBy = ()) 
+                                    returns @tainted stream<File>|error {
+        ListFilesOptional optional = {};
+        string searchString = NAME + SPACE + CONTAINS + SPACE + SINGLE_QUOTE + fileName + SINGLE_QUOTE + SPACE + AND + 
+                                SPACE + TRASH_FALSE;
+        optional.q = searchString;
+        optional.supportsAllDrives = true;
+        optional.includeItemsFromAllDrives = true;
+        if (noOfFiles is int){
+            optional.pageSize = noOfFiles;
+        }
+        if (orderBy is string){
+            optional.orderBy = orderBy;
+        }
+        return getFiles(self.httpClient, optional);
+    } 
+
+    # Retrieve all Google spreadsheets
+    # 
+    # + return - If successful, returns stream of files `stream<File>`. Else returns `error`
+    remote function getAllSpreadsheets() returns @tainted stream<File>|error {
+        ListFilesOptional optional = {};
+        string searchString = TRASH_FALSE + SPACE + AND + SPACE + MIME_TYPE + EQUAL + SHEETS;
+        optional.q = searchString;
+        optional.supportsAllDrives = true;
+        optional.includeItemsFromAllDrives = true;
+        return getFiles(self.httpClient, optional);
+    }
+
+    # Retrieve Google spreadsheets by Name
+    # 
+    # + fileName - Name of the file to search (Partial search)
+    # + noOfFiles - Number of files to retreive 
+    # + orderBy - A comma-separated list of sort keys. Valid keys are 'createdTime', 'folder', 'modifiedByMeTime', 
+    #             'modifiedTime', 'name', 'name_natural', 'quotaBytesUsed', 'recency', 'sharedWithMeTime', 'starred', 
+    #              and 'viewedByMeTime'
+    # + return - If successful, returns stream of files `stream<File>`. Else returns `error`
+    remote function getSpreadsheetsByName(string fileName, int? noOfFiles = (), string? orderBy = ()) 
+                                            returns @tainted stream<File>|error {
+        ListFilesOptional optional = {};
+        string searchString = NAME + SPACE + CONTAINS + SPACE + SINGLE_QUOTE + fileName + SINGLE_QUOTE + SPACE + AND + 
+                                SPACE + TRASH_FALSE + SPACE + AND + SPACE + MIME_TYPE + EQUAL + SHEETS;
```

Comment:
Line breaks should be done like this https://ballerina.io/learn/coding-conventions/#line-breaks

Reply:
Thanks @LakshanSS 
Fixed it